ReactWindows/ReactNative.Shared/Modules/Storage/AsyncStorageHelpers.cs

```diff
@@ -157,5 +158,15 @@ public static JObject GetInvalidValueError(string key)
         {
             return GetError(key, "Invalid Value");
         }
+
+        public static JObject GetError(Exception ex)
+        {
+            return new JObject
+            {
+                { "message", ex.Message },
+                { "key", ex.GetType().Name },
```

Comment:
Can you also stash ex.HResult. It is sometimes embedded in the message, but sometimes it is not. Having a HResult makes debugging easy, especially if the bug report is from a machine having a different user locale than what the dev understands